src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    private static final Random random = new Random();
```

Comment:
The 'random' variable should not be static. Static variables can lead to issues in multi-threaded environments and generally suggest a design that is not object-oriented. Consider making this a private instance variable.